modules/dr-restoring-cluster-state.adoc

```diff
@@ -304,6 +306,8 @@ Perform the following step only if you are using `OVNKubernetes` network plugin.
 $ oc delete node <non-recovery-controlplane-host-1> <non-recovery-controlplane-host-2>
 ----
 
+. Optional: For any remaining non-recovery control plane nodes, delete and recreate each non-recovery control plane node.
```

Comment:
Yes, good point. I see that the term is in the existing docs, but I'll ask the experts.

Reply:
The terms "recovery node" and "non-recovery node" are used to differentiate the node where the recovery procedure is being performed (i.e where we run the backup and restore scripts) from the remainder nodes. Un-recoverable vs recoverable implies that only one node is recoverable so that doesn't seem quite right.

Unless there is a better phrasing that comes to mind, I would stick with recovery/non-recovery for now.

Reply:
Thanks for the reply @hasbro17 . Here is the IBM guidance that writers must adhere to, typically for translation purposes:

![Screenshot from 2023-06-14 12-27-24](https://github.com/openshift/openshift-docs/assets/57954076/1e5bbce3-90d1-45a1-bb1d-51b717553b14)

I did not think changing "non-recovery" to "unrecoverable" would lead to this differentiation, as the context of the script that does the actionable steps will not change. What if I define the terms "recovery" and "unrecoverable" (based on the recovery script's actions) in the following high-level section?

[About disaster recovery](https://60789--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/about-disaster-recovery.html)

Would you be OK to share a snippet of the recovery script's syntax that shows "recovery" and "non-recovery" entries?

Reply:
This sentence is wrong. **Non-recovery nodes must stay deleted until OVN-Kubernetes control plane is re-deployed or it will cause issues!!!!!!**.

I am opening URGENT bug for this shortly.